regress/unittests/win32compat/string_tests.c

```diff
@@ -0,0 +1,40 @@
+#include "includes.h"
+#include <string.h>
+#include "../test_helper/test_helper.h"
+#include "tests.h"
+
+int retValue;
+
+void
+str_simple_tests()
+{
+	TEST_START("string testcases");
+
+	char *s1 = "test_dir";
+	char *s2 = NULL;
+
+	s2 = strdup(s1);
```

Comment:
Looks like NULL cases are not covered. Ex. strdup(NULL)? Can you add them as well. Expectation is that call does not crash.